hera_cal/abscal_funcs.py

```diff
@@ -1003,11 +1016,31 @@ def fft_dly(vis, wgts=None, df=9.765625e4, medfilt=True, kernel=(1, 11), time_ax
     else:
         vis_smooth = vis
 
-    # fft
-    window = np.repeat(signal.windows.tukey(Nfreqs)[np.newaxis], Ntimes, axis=time_ax)
-    window = np.moveaxis(window, 0, time_ax)
-    window *= wgts
-    vfft = np.fft.fft(vis_smooth * window, axis=freq_ax)
+    # construct window
+    win = np.moveaxis(np.repeat(np.zeros(Nfreqs)[np.newaxis], Ntimes, axis=0), 0, time_ax)
+
+    if edge_cut is not None:
+        assert 2*edge_cut < Nfreqs, "edge_cut cannot be >= Nfreqs/2"
```

Comment:
2*edge_cut < Nfreqs+1 (since you need two frequencies to define a delay).

Reply:
i think you mean 2*edge_cut < Nfreqs - 1

Reply:
Yes, you're right.